templates/core/crowdfunding_donors.html

```diff
@@ -0,0 +1,50 @@
+{% extends 'global/base.html' %}
+{% load staticfiles rev %}
+{% load element_by_counter %}
+
+{% block extra_css %}
+    <link rel="stylesheet" type="text/css" href="{% static_rev 'css/donors.css' %}">
+{% endblock %}
+
+{% block content %}
+    <div class="container">
+        <div class="row">
+            <h2>💖 Special thanks to all our amazing supporters! 💖</h2>
+            <br />
+            <div class="col-md-6">
+                <p>
+                This year (2018) we ran a crowdfunding campaign on Indiegogo's generosity website from
+                the 29th January to the 29th of March with the aim of raising US $10,000. We are truly
+                grateful to all our awesome supporters who helped reach our target goal. We managed to
+                raise a total of $13,314 - 133% of our goal!
+                </p>
+            </div>
+            <div class="col-md-6">
+                <img src="{% static 'img/global/donate/django_crowdfunding_heart.png' %}"></a>
+            </div>
+        </div>
+
+        <br/>
+
+        <div class="row">
+            <h2>Our awesome supporters ✨</h2>
+            {% for donor in donor_list %}
+                 {% if forloop.first %}<div class="row">{% endif %}
```

Comment:
This line is not indented nicely - there is one extra space I think. It's super small thing, but it's nice to have nicely formatted code :D. Let's remove one space before `{%`

Reply:
Fixed indentation.